### DISCUSSION TITLE
error: "ResizeObserver loop completed with undelivered notifications"

### DISCUSSION THREAD

**Discussion by kennsorr** [KevinVandy/material-react-table#1155] [General]
I'm getting an error "ResizeObserver loop completed with undelivered notifications".

The error is thrown when the browser window is resized and `editDisplayMode` is set to 'table' in useMaterialReactTable({}).

Has anyone else encountered this, or know of a possible solution?

**Comment by travismiller** [comment]
We observed this error when using virtualization.

A combination of these two things resolved for us:
- set [`useAnimationFrameWithResizeObserver`](https://tanstack.com/virtual/latest/docs/api/virtualizer#useanimationframewithresizeobserver) to `true`
  - https://github.com/TanStack/virtual/issues/977#issuecomment-2775864047
- override the version of @tanstack/react-virtual to 3.13.12. @material-react-table explicitly [depends on 3.10.6](https://github.com/KevinVandy/material-react-table/blob/f7e3287132f6dd9b70063a428a86fe5df9a8342c/packages/material-react-table/package.json#L119).
